src/commands/hsetxx.json

```diff
@@ -0,0 +1,69 @@
+{
+    "HSETXX": {
+        "summary": "Sets the value of a field in a hash only when the field exist.",
+        "complexity": "O(1)",
+        "group": "hash",
+        "since": "2.0.0",
+        "arity": 4,
+        "function": "hsetxxCommand",
+        "command_flags": [
+            "WRITE",
+            "DENYOOM",
+            "FAST"
+        ],
+        "acl_categories": [
+            "HASH"
+        ],
+        "key_specs": [
+            {
+                "flags": [
+                    "RW",
+                    "INSERT"
```

Comment:
I think it should be "UPDATE" instead of insert. See https://redis.io/docs/reference/key-specs/#logical-operation-flags

Reply:
Sure, you right

Reply:
"RW" - also remove?